docs/core/tools/enable-tab-autocomplete.md

```diff
@@ -8,11 +8,11 @@ ms.date: 12/17/2018
 
 # How to enable TAB completion for the .NET Core CLI
 
-Starting with .NET Core CLI v2.0, support for tab completion is included but disabled by default. This article describes how to enable tab completion for your terminal of choice.
+Starting with .NET Core SDK v2.0, support for tab completion is included but disabled by default. This article describes how to enable tab completion for your terminal of choice.
```

Comment:
It's not exactly disabled by default - the script you run to set it up varies by shell, and we do not have a good way to run shell specific stuff at install. 

"disabled by default" -> "needs to be set up for your terminal" 

Is it "terminal" or "shell"